main.py

```diff
@@ -0,0 +1,74 @@
+import logging
+import sys
+from time import sleep
+
+from thespian.actors import ActorAddress, ActorSystem
+
+from src.utils.address_book import AddressBook
+from src.utils.cli import CLI
+from src.utils.constants import ActorNames, Requests
+
+
+def main():
+    """
+    Starts the Bellboy system by configuring logging, creating an ActorSystem,
+    creating the LeadActor, and asking it to START.
+
+    The lead actor loads all other required actors. The system can be
+    halted with a keyboard interrupt (CTRL+C)
+    """
+
+    log = logging.getLogger("Bellboy")
+    log.info("Starting the Bellboy system")
+
+    # Parse arguments and configure logging.
+    cli = CLI()
+    cli.setup(sys.argv)
+
+    # Import actors here to instantiate logging system correctly.
+    from src.actors.lcd import LiquidCrystalActor
+    from src.actors.lead import LeadActor
+    from src.actors.ultrasonic import UltrasonicActor
+
+    # Initialize the Actor system:
+    system: ActorSystem = ActorSystem(systemBase="multiprocQueueBase")
+    address_book = AddressBook()
```

Comment:
the address book seems to be unnecassirly complicating the code

Reply:
It was mean to be a more robust/read-only thing.

Currently if you look at the startup sequence, the address book is passed as a dictionary, and the object isn't used since I couldn't get it to correctly pass between threads. It would arrive empty.

Reply:
what object do u mean ? the whole address book? lets keep it out for now. having everyone have access to the whole list of actors doesnt seem ideal to me! and doesnt seem necessary atm.